solana/program_v2/packages/client/idl/src/solana_anchor_gateway.ts

```diff
@@ -243,14 +243,14 @@ export type SolanaAnchorGateway = {
           "isSigner": false
         },
         {
-          "name": "splTokenProgram",
```

Comment:
I'm just thinking if that is an unnecessary breaking interface change or NOT. As a named parameter it probably is not, but still confused why it's reconfigured.

Reply:
I am not sure why it regenerated like it did. We only added the mint to the withdraw interface

Reply:
yeah, was just curious. the breaking change comes from the `mint` anyway :)